### ISSUE TITLE
Dash live thumbnails with 3 hours of DVR no supported

### ISSUE THREAD

**Issue by hiren3897** [shaka-project/shaka-player#4230]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**
yes

**What version of Shaka Player are you using?**
v3.2.2

**Can you reproduce the issue with our latest release version?**
yes

**Can you reproduce the issue with the latest code from `main`?**
yes

**Are you using the demo app or your own custom app?**
Demo app

**If custom app, can you reproduce the issue using our demo app?**
n/a

**What browser and OS are you using?**
Chrome, Safari, Edge, FireFox
Windows, Linux, iOS, macOS

**For embedded devices (smart TVs, etc.), what model and firmware version are you using?**
n/a

**What are the manifest and license server URIs?**

here is the test URL for you: https://test-drm.hexaglobe.net/hrathod/new-plugin/dash-live-thumbs.php

**What configuration are you using?  What is the output of `player.getConfiguration()`?**

n/a
**What did you do?**

When you mousemove on the seek-bar the thumbnails are only displayed prior to 30 mins if you go more than 30 mins then the first segment reference thumbnail will only be displayed

**What did you expect to happen?**
To have availability of 3hours of segments, thumbnail references from the manifest.

**What actually happened?**

For the test URL, I have provided you can see that,

It only has a ~1400 segment index for the 3 hours of LIVE
I made a debug

The duration of each thumbnail is 1 in the manifest 
layout: 1x1

when we seek more the 30 mins the referencePosition is 0 and it returns the first segmentIndex reference after. 
But the Dash manifest has  more than that because it is 3 hours of DVR and has approx 10000+ thumbnails 
![WhenReferenceIs0](https://user-images.githubusercontent.com/59652404/168793619-0fdfeaf3-2903-441f-86fa-eecc6829c751.png)

when we seek the request is made with the correct time to the player to return then thumbnail reference but because of not having a segment availability it always returns 0. 
LAST SEGMENT returned when referencePositon is 0

![lastSegmentthumb](https://user-images.githubusercontent.com/59652404/168793726-7266307c-b3eb-46a0-9ebd-4e81650e0205.png)

You can have
![whenWeFindReferencePosition](https://user-images.githubusercontent.com/59652404/168793798-affe5026-ca9d-4255-89e2-2fd832e5d73e.png)
 a look when we find a reference it gets the correct Thumbnails from the segments

**Comment by avelad** [comment]
I haven't had time to look at it. Sorry!

**Comment by hiren3897** [comment]
Okay, Please do let me know when you have time for this! Thanks

**Comment by avelad** [comment]
Can you paste here you `player.getConfiguration()`? Thanks!

I think the problem may be related to the config of `manifest.dash.initialSegmentLimit`

**Comment by hiren3897** [comment]
Hey yes, my `manifest.dash.initialSegmentLimit` was default set to 1000.
As I am having 3 hours of DVR in my live streams so I set it to `manifest.dash.initialSegmentLimit: 10800` and it works very well :)

**Comment by hiren3897** [comment]
But when I try to seek the thumbnails they are not accurate every time. 
when the user hovers it requests the thumbnail with the correct time but the thumbnails are displayed wrong sometimes. :/

**Comment by hiren3897** [comment]
I did the modifications in the same demo: https://test-drm.hexaglobe.net/hrathod/new-plugin/dash-live-thumbs.php

you can also check the `player.getConfiguration()` in the console

**Comment by hiren3897** [comment]
I think the reason could be the latency of the stream? no?

**Comment by avelad** [comment]
Ok the main problem that was reported in this issue is resolved. As for the other issue, I think it's a latency issue and Live's playback window continues (Live stream).

**Comment by hiren3897** [comment]
Yes, I will dig into some research for the accuracy of thumbnails on live streams. Thanks for your help.
